### PR TITLE
Added Template to create New Slice

### PR DESCRIPTION
**Pull request by Kamini-Mysha**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by satravi** [comment]
Azure deploy parameters file to be added.

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/prereqs/prereq.azuredeploy.json [hunk 1]
```diff
@@ -35,18 +35,6 @@
         "description": "The mobile network code for the private mobile network"
       }
     },
-    "sliceName": {
```
Comment (satravi) [inline quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/prereqs/prereq.azuredeploy.json]: Why the slice and sst parameters are removed from prereq Mobile Network?

**Comment by bmoore-msft** [comment]
@Kamini-Mysha are you able to address @satravi's comments?

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/prereqs/prereq.azuredeploy.json (hunk 1 above)
Comment (Kamini-Mysha) [reply quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/prereqs/prereq.azuredeploy.json]: we want the quickstart templates to be a fast way to setup a basic mobile network -we're considering multiple slices/ non-default slice as non-basic

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/main.bicep [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+resource existingMobileNetwork 'Microsoft.MobileNetwork/mobileNetworks@2022-04-01-preview' existing = {
+  name: existingMobileNetworkName
+
+  #disable-next-line BCP081
+  resource exampleSlice 'slices@2022-04-01-preview' = {
+    name: sliceName
+    location: location
+    properties: {
+      snssai: {       
+        sst: sst
```
[the remark this answers is not in the document]
Comment (Kamini-Mysha) [reply quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/main.bicep:28]: Done

**Review by satravi — APPROVED** [review]

**Review by tomzhu1 — APPROVED** [review]

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/azuredeploy.parameters.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        "existingMobileNetworkName": {
+            "value": "GET-PREREQ-existingMobileNetworkName"
+        },
+        "sliceName": {
+            "value": "GEN-UNIQUE"
+        },
+        "sst": {
+            "value": 1
+        },
+        "sd": {
```
Comment (bmoore-msft) [inline quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/azuredeploy.parameters.json:17]: remove this so the defaultValue can be tested

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/prereqs/prereq.azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
```
Comment (bmoore-msft) [inline quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/prereqs/prereq.azuredeploy.json:5]: if this is generated by bicep, include only the bicep source file...

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/main.bicep [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+@description('The SST value for the slice being deployed.')
+@maxValue(255)
+@minValue(0)
+param sst int
+
+@description('The SD value for the slice being deployed.')
+param sd string=''
+
+#disable-next-line BCP081
```
Comment (bmoore-msft) [inline quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/main.bicep:18]: remove directive

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/main.bicep [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+resource existingMobileNetwork 'Microsoft.MobileNetwork/mobileNetworks@2022-04-01-preview' existing = {
+  name: existingMobileNetworkName
+
+  #disable-next-line BCP081
```
Comment (bmoore-msft) [inline quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/main.bicep:22]: remove directive

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/prereqs/prereq.azuredeploy.json (hunk 4 above)
Comment (Kamini-Mysha) [reply quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/prereqs/prereq.azuredeploy.json:5]: This is a Prerequisite file and is not generated by bicep File

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/azuredeploy.parameters.json (hunk 3 above)
Comment (Kamini-Mysha) [reply quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/azuredeploy.parameters.json]: Done

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/main.bicep (hunk 6 above)
Comment (Kamini-Mysha) [reply quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/main.bicep]: Done

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/main.bicep (hunk 5 above)
Comment (Kamini-Mysha) [reply quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/main.bicep]: Done

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/prereqs/prereq.azuredeploy.json (hunk 4 above)
Comment (bmoore-msft) [reply quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/prereqs/prereq.azuredeploy.json:5]: The metadata property has the bicep header in it... so this file, at some point, was generated by bicep or someone manually authored it to look that way (which would be odd but anyway)...

if there's bicep source for the file add it to the PR and remove the json.  If there's no bicep source, remove the metadata.

Comment (Kamini-Mysha) [reply quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/prereqs/prereq.azuredeploy.json]: Removed Metadata

**Pull request merged**